### ISSUE TITLE
using jest and ts-jest with express-zod-api

### ISSUE THREAD

**Issue by John4266** [RobinTail/express-zod-api#551]
Hello there.
I am trying to setup a project and have tests with `jest` and `ts-jest`.
The installation of `jest` itself is pretty smooth.
However, whenever I try to install `ts-jest`, i get the following error:
![image](https://user-images.githubusercontent.com/70203256/185181576-909da99d-7797-4acb-8f7b-b0e8d4ffefeb.png)
![image](https://user-images.githubusercontent.com/70203256/185181659-a33422ea-edc8-4011-baf7-918ba032cb4d.png)

Is it possible to install `ts-jest` along with the `express-zod-api` package without forcing it with either of the `--force, or --legacy-peer-deps` flags?

P.S., Here is some info about my environment:
```
OS: Windows 10 | 64 bit
Node.js Version: 16.17.0
npm Version: 8.17.0
```

**Comment by RobinTail** [comment]
thank you for reporting this @John4266 .

The issue is caused by the following line in the package.json:
```
    "jest": ">=25 <28",
```

since I did not test the provided `testEndpoint` function on jest 28.

I'm going to fix it.

**Comment by RobinTail** [comment]
🚀  The issue should be fixed in recently released version 7.8.1, @John4266 .
In case it's not, please inform me by creating a new issue.
Thank you.
